applications/main/gpio/scenes/gpio_scene_usb_uart_config.c

```diff
@@ -116,7 +116,12 @@ void gpio_scene_usb_uart_cfg_on_enter(void* context) {
     variable_item_set_current_value_index(item, cfg_set->uart_ch);
     variable_item_set_current_value_text(item, uart_ch[cfg_set->uart_ch]);
 
-    item = variable_item_list_add(var_item_list, "RTS/DTR Pins", 3, line_flow_cb, app);
+    // GPIO pins PC0, PC1 (16,15) are unavailable for RTS/DTR when LPUART is
+    // selected.
+    int available_flow_pins = cfg_set->uart_ch == FuriHalUartIdLPUART1 ? 3 : 4;
```

Comment:
Nope. That's not enough.
See line_flow_cb and line_port_cb

Reply:
Ensure that when you go to configuration it is impossible to set invalid combination.